utils/yoke/src/yoke.rs

```diff
@@ -443,9 +443,6 @@ impl<Y: for<'a> Yokeable<'a>, C: StableDeref> Yoke<Y, Option<C>> {
     /// this is good for e.g. constructing fully owned
     /// [`Yoke`]s with no internal borrowing.
     ///
-    /// This can be paired with [`Yoke::attach_to_option_cart()`] to mix owned
```

Comment:
@Manishearth is there a replacement for this?

Reply:
`wrap_cart_in_option()`